File: Timw44/src/FizzBuzz.java
public class FizzBuzz
{
    public static void main(String[] args)
    {
        String fizz = "Fizz";
        String buzz = "Buzz";

        for(int i = 1; i <= 100; i++)
        {
            if(i % 3 == 0 && i % 5 == 0)
            {
                System.out.println(fizz + buzz);
            }
            else if(i % 5 == 0)
            {
                System.out.println(buzz);
            }
            else if(i % 3 == 0)
            {
                System.out.println(fizz);
            }
            else
            {
                System.out.println(i);
            }
        }
    }
}

File: Twesha/fizzbuzz.cpp
#include<iostream>
using namespace std;

int main()
{
	for(int i=1;i<=100;i++){
		if(i%3==0 && i%5==0) cout<<"fizzbuzz ";
		else if(i%3==0) cout<<"fizz ";
		else if(i%5==0) cout<<"buzz ";
		else cout<<i<<ends;
	}
	
}
